src/index.html

```diff
@@ -1,14 +1,25 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8" />
     <title>Dockstore</title>
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="../assets/favicon.ico" />
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons&display=swap" rel="stylesheet" />
     <link rel="manifest" href="manifest.json" />
     <meta name="theme-color" content="#21335B" />
+    <meta
+      name="Description"
+      content="Dockstore, developed by the Cancer Genome Collaboratory, is an open platform used
+     by the GA4GH for sharing Docker-based tools described with the Common Workflow Language (CWL), the Workflow
+     Description Language (WDL), or Nextflow (NFL)"
+    />
+
+    <!--As suggested by Google Lighthouse audit, preload fonts-->
```

Comment:
In my tests a while back, this looked like the most "bang-for-the-buck" so this deserves a longer look/investigation.

Reply:
There's probably something i'm missing, as preloading and `font-display: swap` are supposed to go nicely together, but i'm out of ideas